recipe/install_llvm.bat

```diff
@@ -7,5 +7,19 @@ if "%PKG_NAME%" == "llvm-tools" (
     rmdir /S /Q %LIBRARY_LIB%
     rmdir /S /Q %LIBRARY_INC%
     rmdir /S /Q %LIBRARY_PREFIX%\libexec
+) else (
+    REM upstream picks up diaguids.lib from the windows image, see
+    REM https://github.com/llvm/llvm-project/blob/llvmorg-14.0.6/llvm/lib/DebugInfo/PDB/CMakeLists.txt#L17
+    REM which ultimately derives from VSINSTALLDIR, see
+    REM https://github.com/llvm/llvm-project/blob/llvmorg-14.0.6/llvm/cmake/config-ix.cmake#L516
+    REM and gets hardcoded by CMake to point to the path in our windows image.
+    REM This makes it non-portable between image versions (e.g. 2019 vs 2022), so replace
+    REM the hardcoded path with a variable again
+    sed -i 's#C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise#%AN_UNSET_VARIABLE_TO_KEEP_THE_PERCENT_SIGNS%#g' %LIBRARY_LIB%\cmake\llvm\LLVMExports.cmake
+    REM two-step process to ensure we have %VSINSTALLDIR% in the end even if it's set already
+    REM (escaping % with %% doesn't seem to work with sed; variable value still gets inserted)
+    sed -i 's#AN_UNSET_VARIABLE_TO_KEEP_THE_PERCENT_SIGNS#VSINSTALLDIR#g' %LIBRARY_LIB%\cmake\llvm\LLVMExports.cmake
```

Comment:
WTH, I tested this locally (in a `cmd.exe`), it's not using regex, why would this be sensitive to spaces or slashes?

```
[0/1] Install the project...
/Microsoft was unexpected at this time.
```